api/swagger/swagger.yaml

```diff
@@ -147,31 +147,50 @@ paths:
           description: Error
           schema:
             $ref: "#/definitions/ErrorResponse"
-  /event/{page},{pageSize}:
-      x-swagger-router-controller: EventController
-      get:
-        description: get a page of events
-        operationId: getEvents
-        parameters:
-          - name: page
-            type: number
-            in: path
-            required: true
-          - name: pageSize
-            type: number
-            in: path
-            required: true
-            default: 10
-        responses:
-          "200":
-            description: Success
-            schema:
-              $ref: "#/definitions/EventListResponse"
-          # responses may fall through to errors
-          default:
-            description: Error
-            schema:
-              $ref: "#/definitions/ErrorResponse"
+  /events/:
+    x-swagger-router-controller: EventController
+    get:
+      description: get pagenated events within a radius or by created time if user location is not provided
+      operationId: getEvents
+      parameters:
+        - name: sortBy
+          type: string
+          in: header
```

Comment:
those params should be in path

Reply:
/events/{sortBy},{page},{pageSize},{lat},{lng},{radius}:

like this?

Reply:
not sure about the swagger format, but the request would be like `/events?page=2&page_size=20&lat=47.3333&lng=102.333433&radius=1000`